components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/dto/TierDTO.java

```diff
@@ -23,6 +23,12 @@
   
   private String description = null;
   
+  public enum TierLevelEnum {
+     api,  application,  resource, 
```

Comment:
Are you saying its a problem to have it in upper case? If not we should stick with upper case.